tests/layer_tests/pytorch_tests/test_unfold.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+import numpy as np
+import torch
+
+
+@pytest.mark.parametrize('dimension', (0, 1, 2))
+@pytest.mark.parametrize('size', (1, 2))
+@pytest.mark.parametrize('step', (1, 2))
+@pytest.mark.parametrize('input_tensor', (np.random.randn(2, 2, 5).astype(np.float32),
+                                          np.random.randn(3, 3, 3, 3).astype(np.float32)))
```

Comment:
please try to use shapes with different dimensions like `(2,3,4,5)`

Reply:
done